src/main/java/apoc/custom/CypherProcedures.java

```diff
@@ -589,30 +655,31 @@ public void unavailable() {
         public static Map<String, Object> storeProcedure(GraphDatabaseAPI api, String name, String statement, String mode, List<List<String>> outputs, List<List<String>> inputs, String description) {
 
             Map<String, Object> data = map("statement", statement, "mode", mode, "inputs", inputs, "outputs", outputs, "description", description);
-            return updateCustomData(getProperties(api), name, PROCEDURES, data);
+            return updateCustomData(api, name, PROCEDURES, data);
         }
 
         public static Map<String, Object> storeProcedure(GraphDatabaseAPI api, ProcedureSignature signature, String statement) {
             Map<String, Object> data = map("statement", statement, "mode", signature.mode().toString(), "signature", signature.toString(), "description", signature.description().orElse(null));
-            return updateCustomData(getProperties(api), signature.name().toString(), PROCEDURES, data);
+            return updateCustomData(api, signature.name().toString(), PROCEDURES, data);
         }
         public static Map<String, Object> storeFunction(GraphDatabaseAPI api, String name, String statement, String output, List<List<String>> inputs, boolean forceSingle, String description) {
             Map<String, Object> data = map("statement", statement, "forceSingle", forceSingle, "inputs", inputs, "output", output, "description", description);
-            return updateCustomData(getProperties(api), name, FUNCTIONS, data);
+            return updateCustomData(api, name, FUNCTIONS, data);
         }
 
         public static Map<String, Object> storeFunction(GraphDatabaseAPI api, UserFunctionSignature signature, String statement, boolean forceSingle) {
             Map<String, Object> data = map("statement", statement, "forceSingle", forceSingle, "signature", signature.toString(), "description", signature.description().orElse(null));
-            return updateCustomData(getProperties(api), signature.name().toString(), FUNCTIONS, data);
+            return updateCustomData(api, signature.name().toString(), FUNCTIONS, data);
         }
 
         public synchronized static Map<String, Object> remove(GraphDatabaseAPI api, String name, String type) {
-            return updateCustomData(getProperties(api),name, type,null);
+            return updateCustomData(api,name, type,null);
         }
 
-        private synchronized static Map<String, Object> updateCustomData(GraphProperties properties, String name, String type, Map<String, Object> value) {
+        private synchronized static Map<String, Object> updateCustomData(GraphDatabaseAPI api, String name, String type, Map<String, Object> value) {
             if (name == null || type==null) return null;
-            try (Transaction tx = properties.getGraphDatabase().beginTx()) {
+            GraphProperties properties = getProperties(api);
+            try (Transaction tx = api.beginTx()) {
```

Comment:
what's the difference between this and the previous version?

Reply:
reverted was related to a change that I reverted